apps/browser/src/content/sites/_common/types.ts

```
@@ -1,11 +1,9 @@
-import type { BrowserObj } from '@eurora/browser-shared/content/extensions/watchers/watcher';
+import type { BrowserObj } from '../../../shared/content/extensions/watchers/watcher';
 
 export type CommonMessageType = 'GET_METADATA';
 
 // eslint-disable-next-line @typescript-eslint/no-empty-object-type
-export interface WatcherParams {
-	// Common watcher doesn't need specific parameters for now
-}
+export interface WatcherParams {}
```

Biome (2.3.14)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)